src/bindings/python/src/pyopenvino/graph/ops/util/multisubgraph.hpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include "openvino/op/util/sub_graph_base.hpp"
+
+namespace py = pybind11;
+
+using MultiSubgraphInputDescriptionVector = ov::op::util::MultiSubGraphOp::MultiSubgraphInputDescriptionVector;
+using MultiSubgraphOutputDescriptionVector = ov::op::util::MultiSubGraphOp::MultiSubgraphOutputDescriptionVector;
+namespace MultiSubgraphOp {
```

Comment:
Shouldn't it have one more namespace around it? `ov::`

Reply:
It should not. There are only helpers in this namespace. I will rename it to `MultiSubgraphHelpers` so it does not interfere with original name.